src/jllvm/materialization/CodeGeneratorUtils.cpp

```diff
@@ -77,8 +77,394 @@ class TrivialDebugInfoBuilder
         m_debugBuilder.finalize();
     }
 };
+
+inline bool isCategoryTwo(llvm::Type* type)
```

Comment:
```suggestion
bool isCategoryTwo(llvm::Type* type)
```
No need for `inline` in a cpp file